platform/javascript/javascript_singleton.cpp

```diff
@@ -285,7 +285,7 @@ Ref<JavaScriptObject> JavaScript::get_interface(const String &p_interface) {
 	return Ref<JavaScriptObject>(memnew(JavaScriptObjectImpl(js_id)));
 }
 
-Variant JavaScript::_create_object_bind(const Variant **p_args, int p_argcount, Callable::CallError &r_error) {
+Ref<JavaScriptObject> JavaScript::_create_object_bind(const Variant **p_args, int p_argcount, Callable::CallError &r_error) {
```

Comment:
This should probably stay `Variant`, since `_js2variant` returns a variant.

Reply:
that's a mistake, should be good now